config.yaml

```diff
@@ -164,6 +158,13 @@ client:
         #     Content-Type: "application/json"
         #     Auth-Token: "12345"
     odp:
+      ## Disable odp
+      disable: false
+      ## Max number of items that can be stored in odp segments cache.
+      segmentsCacheSize: 10000
+      ## Timeout in seconds after which an item stored in odp segments cache will be deleted.
+      segmentsCacheTimeout: 600s
+      ## This will override all other settings
       cache:
```

Comment:
Looks good! What about "segmentsCache" instead of "cache" to be consistent with "segmentsCacheSize" and "segmentsCacheTimeout"?

Reply:
sure, will update this in the parent PR for odp-cache, this is branched off of it.